source/dub/commandline.d

```diff
@@ -216,7 +216,7 @@ int runDubCommandLine(string[] args)
 			dub.defaultPlacementLocation = options.placementLocation;
 		} else {
 			// initialize DUB
-			auto package_suppliers = options.registry_urls.map!(url => cast(PackageSupplier)new RegistryPackageSupplier(URL(url))).array;
+			auto package_suppliers = options.registry_urls.map!(url => cast(PackageSupplier)new RegistryPackageSupplier(URL(url.chomp("/")~"/"))).array;
```

Comment:
Perhaps just something minor, but if the trailing slash is already there, this isn't the most efficient implementation.